src/main/java/uk/gov/hmcts/juror/job/execution/config/SmtpConfig.java

```diff
@@ -1,28 +0,0 @@
-package uk.gov.hmcts.juror.job.execution.config;
-
-import jakarta.validation.constraints.Email;
-import jakarta.validation.constraints.NotBlank;
-import jakarta.validation.constraints.NotNull;
-import lombok.Getter;
-import lombok.Setter;
-import org.springframework.boot.context.properties.ConfigurationProperties;
-import org.springframework.context.annotation.Configuration;
-
-@Configuration
-@ConfigurationProperties(prefix = "defaults.smtp")
-@Getter
-@Setter
-public class SmtpConfig {
-
-    @Email
-    @NotBlank
```

Comment:
We will want to keep this class in case we need it for the future

Reply:
As requested, change reverted.  However dead code should not be in live code.  If the code is required in the future, it can be retrieved from source control.